api/src/main/java/io/minio/MinioRequestBody.java

```diff
@@ -0,0 +1,66 @@
+package io.minio;
```

Comment:
Add the license header just like other files... @tobias-

Reply:
Done.